theme/src/components/nav-items.js

```diff
@@ -1,5 +1,6 @@
+import {FocusKeys} from '@primer/behaviors'
 import {LinkExternalIcon} from '@primer/octicons-react'
-import {NavList} from '@primer/react/drafts'
+import {NavList,useFocusZone} from '@primer/react'
```

Comment:
I _think_ we'll need to bump the version of `@primer/react` in `theme` to a newer version as right now it's at `35.2.2` and I think this is only available in later versions

Reply:
That would be ideal, I did it locally and the update breaks some stuff that needs to be adapted. The changes on this PR are independent on which version for `>=35.2.2`. Don't take me wrong, the version should be updated, but not related with the work in this PR.